app/src/main/java/edu/uc/kansakpb/ikeahjalpare3/dto/Furniture.kt

```diff
@@ -4,13 +4,12 @@ import com.google.gson.annotations.SerializedName
 
 data class Furniture(
```

Comment:
Changed the order of the class attributes so that when they are displayed the order reflects the needs of the user interface, for presentation purposes.

Reply:
It looks like you want to change the order of parameters in the toString(), which is fine.... but you don't need to change the order of the parameters in the constructor as well.  If this is all fed from JSON, it's probably OK, but if the constructor is invoked elsewhere, the parameter values will no longer match the parameter variables, meaning you might have video link stored in article_number.

Also, fix the spelling of Manual while you are here.